examples/response/vp_token_raw_ldp_vp.json

```diff
@@ -40,7 +40,7 @@
     "type": "Ed25519Signature2018",
     "created": "2021-03-19T15:30:15Z",
     "challenge": "n-0S6_WzA2Mj",
-    "domain": "https://client.example.org/cb",
+    "domain": "x509_san_uri:https://client.example.org/cb",
```

Comment:
I don't think I have the necessary knowledge, can someone confirm that this change is correct please? It looks at best odd to combine x509_san_uri with did holder & did verification method.

Reply:
I don't honestly remember why that seemed like the right/best thing to change there and [vp_token_multiple_vps.json](https://github.com/openid/OpenID4VP/pull/401/files#diff-ef8c16339812a418241f69e9dcdafb60a1fc89dc227b1bbb26ff362d76b1d15fR44) above which appear in https://openid.net/specs/openid-4-verifiable-presentations-1_0-24.html#section-8.1.2 but it being openid federation (b/c there's no prefix prior to this PR and that's the special case) didn't seem right either. Maybe `redirect_uri:` would make more sense?

Reply:
That seems fair. And I since discovered that the `domain` claim is actually explicitly defined to be the client_id ( https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#appendix-B.1.2.3 ) so I agree this change is at least not introducing a new problem.